src/components/playground/CustomizerSlider.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Radium from 'radium';
+
+@connect(state => state.playground)
+@Radium
+class CustomizerSlider extends React.Component {
```

Comment:
I would say hang onto it if you want to, but let's take it out of source control for now until there's a need for it.